lib/all-badge-examples.js

```diff
@@ -647,6 +647,34 @@ const allBadgeExamples = [
           'node'
         ]
       },
+      {
+        title: 'custom registry :: npm',
+        previewUri: '/npm/https/registry.npm.taobao.org/v/npm.svg',
+        keywords: [
+          'node'
+        ]
+      },
+      {
+        title: 'custom registry :: npm (scoped)',
+        previewUri: '/npm/https/registry.npm.taobao.org/v/@cycle/core.svg',
+        keywords: [
+          'node'
+        ]
+      },
+      {
+        title: 'custom registry :: npm (tag)',
+        previewUri: '/npm/https/registry.npm.taobao.org/v/npm/next.svg',
+        keywords: [
+          'node'
+        ]
+      },
+      {
+        title: 'custom registry :: npm (scoped with tag)',
+        previewUri: '/npm/https/registry.npm.taobao.org/v/@cycle/core/canary.svg',
```

Comment:
How about choosing one of these four and removing the others? The title should be something like **npm (scoped with tag, custom registry)**. Same with the node badges. I feel like people will understand the pattern.

Reply:
![screenshot-2017-11-19 shields io quality metadata badges for open source projects 2](https://user-images.githubusercontent.com/559321/32987286-74c64bb8-cce7-11e7-8a79-748aa6b1f469.png)
![screenshot-2017-11-19 shields io quality metadata badges for open source projects 1](https://user-images.githubusercontent.com/559321/32987290-7be76ddc-cce7-11e7-94a4-f2678792c614.png)

Reply:
it looks like this now, hope its okay